code/src/index.css

```diff
@@ -1,13 +1,166 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: "Chakra Petch", sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: rgb(10, 37, 255);
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.mainContainer {
+  background-color: rgb(255, 236, 234);
+  border: 2px solid rgb(246, 166, 34);
+  width: 320px;
+  margin: 50px auto;
+  padding: 10px;
+}
+
+h1 {
+  padding-top: 60px;
+  font-size: 18px;
+  width: fit-content;
+  margin: auto;
+  color: rgb(254, 53, 37);
+}
+
+h2 {
+  padding-top: 10px;
+  font-size: 18px;
+  width: fit-content;
+  margin: auto;
+  color: rgb(254, 53, 37);
+}
+
+h3 {
+  margin: 20px auto 40px;
+  width: fit-content;
+  display: flex;
+  font-size: 18px;
+  color: rgb(254, 53, 37);
+}
+
+p {
+  font-family: "Chakra Petch", sans-serif;
+  width: 320px;
+  max-width: 400px;
+  color: rgb(254, 53, 37);
+}
+
+.loader {
```

Comment:
The animation is positioned very close (almost above) to the pink container. Maybe move it down a bit to get it more visible?